foundry.toml

```diff
@@ -5,7 +5,7 @@
   block_timestamp = 1_680_220_800 # March 31, 2023 at 00:00 GMT
   bytecode_hash = "none"
   evm_version = "shanghai"
-  fuzz = { runs = 1_000 }
+  fuzz = { runs = 1_000_00 }
```

Comment:
This takes 1.5 sec to be executed on my machine. If you prefer we can keep it smaller.

Reply:
This is fine by me